src/main/resources/base-target.yaml

```diff
@@ -138,7 +138,9 @@ target:
             - video/*
           # Look into all XML descriptors to index all binary files referenced
           metadataPathPatterns:
-            - ^/?site/.+\.xml$
+            - ^/(?!config).+\.xml$
```

Comment:
Isn't this going to process ALL XML files as metadata files in the repo? I know it's needed in authoring, but this will be done in delivery too as it is now.

Reply:
Maybe authoring should have its own set of properties?

Reply:
Those are already only for authoring

Reply:
oh sorry didn't see that